### PR TITLE
✨ Feature/clear all categories

### PR DESCRIPTION
**Pull request by tykayn**
adds a button to clear all selected categories, displaying translated title for the button.
Screenshot 2023-01-31 at 14-25-22 Carte Touristique des Landes - Carte Touristique des Landes

before:
Screenshot 2023-01-27 at 17-11-14 Carte Touristique des Landes - Carte Touristique des Landes

small display: 
Screenshot 2023-01-30 at 11-54-10 Carte Touristique des Landes - Carte Touristique des Landes

Screenshot 2023-01-30 at 11-53-56 Carte Touristique des Landes - Carte Touristique des Landes

### CODE REVIEW

File: components/Home/Home.vue [hunk 1]
```diff
@@ -149,7 +150,7 @@
               @click="onBottomMenuButtonClick"
             >
               <span class="sr-only">{{ $tc('headerMenu.categories') }}</span>
-              <font-awesome-icon icon="grip-lines" size="lg" />
+              <font-awesome-icon icon="grip-lines" />
```
Comment (frodrigo) [inline components/Home/Home.vue:153]: Required changes ?

File: components/Home/Home.vue [hunk 2]
```diff
@@ -160,7 +161,6 @@
           <font-awesome-icon
             icon="spinner"
             class="text-zinc-400 animate-spin"
-            size="3x"
```
Comment (frodrigo) [inline components/Home/Home.vue:163]: Required changes ?

File: locales/es.js [hunk 3]
```diff
@@ -67,6 +67,7 @@ export default {
     cartocode: 'Cartocode',
     noResult: 'No hay resultados',
     hideCategory: 'Ocultar esta categoría',
+    clearAllCategories: 'Ocultar todo los categoría',
```
Comment (frodrigo) [inline locales/es.js:70]: Not really Spanish. Just omiit Spanish if you cannot wrtie it. I will do.
```suggestion
    clearAllCategories: 'Ocultar todas las categorías',
```

File: components/Home/Home.vue [hunk 4]
```diff
@@ -617,6 +617,10 @@ export default (
         (categoryId) => !categoriesIds.includes(categoryId)
       )
     },
+    clearAllCategories() {
+      console.log('this.selectedCategoriesIds', this.selectedCategoriesIds)
```
Comment (frodrigo) [inline components/Home/Home.vue:621]:
```suggestion
```

File: components/Home/SelectedCategories.vue [hunk 5]
```diff
@@ -1,8 +1,8 @@
 <template>
   <div
     v-if="selectedMenuItems.length > 0"
-    class="flex flex-row p-2 flex-wrap bg-white shadow-md pointer-events-auto rounded-xl max-w-xl"
-    style="min-width: 64px"
+    class="list-of-categories-selected flex flex-row p-2 flex-wrap bg-white shadow-md pointer-events-auto rounded-xl max-w-sm min-w-xs"
+    :class="{ 'scroll-pr-2': selectedMenuItems.length > 1 }"
```
Comment (frodrigo) [inline components/Home/SelectedCategories.vue:5]: To check

**Comment by tykayn** [comment]
adds a clear categories button 

Screenshot 2023-01-27 at 17-11-14 Carte Touristique des Landes - Carte Touristique des Landes

**Comment by frodrigo** [comment]
> selected categories disappear on mobile display, is it a problem ?

No. It is on purpose.

File: components/Home/SelectedCategories.vue (hunk 5 above)
Comment (tykayn) [reply components/Home/SelectedCategories.vue]: removed unused class, conditionnal class works fine

**Comment by frodrigo** [comment]
Could you rebase on dev2, please.

**Comment by tykayn** [comment]
> Could you rebase on dev2, please.

yes, i found the clear button had a missing aria label to please the html validation in cypress. now it works fine.

added some screenshots in first message.

**Comment by frodrigo** [comment]
I found the button ugly:
- Should be red like the others.
- What about put it on the right top corner, not at an intermediate location.

**Comment by frodrigo** [comment]
This branch contain too many commits and other features. Please rebase the branch on top of dev2 (`git rebase dev2`).

**Comment by frodrigo** [comment]
Target is now `develop` branch.

**Comment by frodrigo** [comment]
> [Merge branch 'develop' into feature/clear-all-categories](https://github.com/teritorio/vido/pull/53/commits/617f4ef5de4692402e759816d0276f7e56cd80ad) https://github.com/teritorio/vido/pull/53

On PR brach you should not merge, only rebase. Again there is unwanted commit / diff on the PR.

**Comment by frodrigo** [comment]
I clean the branch, the commit, and merged manually
d0a33ae90758684aecbc4455f12426b1a29add54

**Pull request closed without merging**